server/test/unit/test_repo_distributor_manager.py

```diff
@@ -237,7 +237,7 @@ def test_add_distributor_validate_raises_error(self):
         try:
             self.distributor_manager.add_distributor('rohan', 'mock-distributor', {}, True)
             self.fail('Exception expected')
-        except exceptions.PulpDataException:
+        except Exception, e:
```

Comment:
The reason for this is that the exception that bubbles up is a generic Exception. Previously we caught this and reraised more specific exception.